package.json

```diff
@@ -16,9 +16,29 @@
   "main": "lib/module.js",
   "scripts": {
     "lint": "eslint --ext .vue,.js lib test",
-    "release": "standard-version && git push --follow-tags && npm publish",
+    "release": "release-it",
+    "release-legacy": "standard-version && git push --follow-tags && npm publish",
     "test": "npm run lint && jest"
   },
+  "release-it": {
```

Comment:
I would suggest moving it to `.release-it.js`:

- This configuration does not needs to be published with package. `package.json` is always in NPM package
- It is easier to update and maintain to be consistent across all module repos -- we can even create a preset in future